### PR TITLE
client: add pl-Pl translations

### PR DESCRIPTION
**Pull request by chappjc**
Moved and tweaked commits from https://github.com/decred/dcrdex/pull/1421 on to `master`.  All commits still have you as the author @artikozel.

### CODE REVIEW

**Comment by chappjc** [comment]
Devs can put up screen shots as we get to this.  Will try to do so on Monday.  Would be awesome to squeeze this in with 0.4.  Thanks a ton @artikozel !

**Comment by JoeGruffins** [comment]
Can't read polish (pl is polish right?) but looks good to me.
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

**Review by JoeGruffins — APPROVED** [review]

File: client/webserver/locales/pl-pl.go [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+	"All markets at":              "Wszystkie rynki na",
+	"pick a different asset":      "wyierz inne aktywo",
+	"Register_loudly":             "Zarejestruj!",
+	"1 Sync the Blockchain":       "1: Zsynchronizuj blockchain",
+	"Progress":                    "Postęp",
+	"remaining":                   "pozostało",
+	"2 Fund the Registration Fee": "2: Wnieś opłatę rejestracyjną",
+	"Registration fee":            "Opłata rejestracyjna",
+	"Your Deposit Address":        "Twój adres do wpłaty",
+	"add a different server":      "dodaj inny serwer",
```
Comment (chappjc) [inline client/webserver/locales/pl-pl.go:187]: @artikozel On master there's one right between the two above:

```diff
	"Your Deposit Address":        "Your Deposit Address",
+	"Send enough for reg fee":     `Make sure you send enough to also cover network fees.`,
	"add a different server":      "add a different server",
```

@JoeGruffins posted a screenshot of this here: https://github.com/decred/dcrdex/pull/1422#issuecomment-1014572770

Comment (chappjc) [reply client/webserver/locales/pl-pl.go:187]: Oh, @artikozel, you're welcome to just provide edits in comments on Github.  Alternatively, if you get the git business all handled, you can get a branch up to replace this PR.  Your call.  No sweat either way.

**Comment by JoeGruffins** [comment]
left out buy and sell
limit buy and sell
image
image
market buy
image

File: client/webserver/locales/pl-pl.go (hunk 1 above)
Comment (artikozel) [reply client/webserver/locales/pl-pl.go:187]: @chappjc
"Send enough for reg fee":     `Upewnij się, że wysyłasz wystarczająco dużo środków, aby pokryć również opłaty sieciowe.`

Still fighting github. This way's dirtier, but speed is of key improtance here :)

**Comment by artikozel** [comment]
> left out buy and sell limit buy and sell 
I'm relatively sure last time I checked in with @chappjc he said these were hardcoded and it would be best to just leave them out. If something's changed in that regard, I'll be happy to address it.

Just in case:
limit buy order: "Zlecenie kupna (limit)"
limit sell order: "Zlecenie sprzedaży (limit)"

File: client/webserver/locales/pl-pl.go [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+	"Add a custom server":         "Dodaj niestandardowy serwer",
+	"plus tx fees":                "+ opłaty transakcyjne",
+	"Export Seed":                 "Eksportuj ziarno",
+	"Total":                       "W sumie",
+	"Trading":                     "Wymiana",
+	"Receiving Approximately":     "Otrzymując około",
+	"Fee Projection":              "Szacunkowa opłata",
+	"details":                     "szczegóły",
+	"to":                          "to",
```
Comment (artikozel) [inline client/webserver/locales/pl-pl.go:196]: Completely missed this one, sorry. Should be "do" instead of the original "to".

**Comment by JoeGruffins** [comment]
@artikozel I meant I left out the pics, not anything you did. Good work though!

**Review by buck54321 — APPROVED** [review]

**Pull request merged**